edgeless_telemetry/src/elasticsearch_api.rs

```diff
@@ -0,0 +1,203 @@
+use chrono::{DateTime, Utc};
+use elasticsearch::indices::IndicesCreateParts;
+use elasticsearch::{
+    auth::Credentials, http::transport::SingleNodeConnectionPool, http::transport::TransportBuilder, Elasticsearch, IndexParts, SearchParts,
+};
+use serde_json::json;
+use serde_json::Value;
+use url::Url;
```

Comment:
We don't import (`use`), but instead prefix them, e.g. in the code below you would use `url:Url`